sources/packages/backend/workflow/test/2021-2022/parttime-assessment/costs/parttime-assessment-2021-2022-additional-transportation-costs.e2e-spec.ts

```diff
@@ -0,0 +1,370 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+import { OfferingDeliveryOptions } from "@sims/test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-additional-transportation-costs.`, () => {
+  it(
+    "Should determine calculatedDataTotalTransportationAllowance when studentDataAdditionalTransportKm is zero " +
+      "and offeringDelivered is onsite",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataAdditionalTransportKm = 0;
+      assessmentConsolidatedData.offeringWeeks = 30;
+      assessmentConsolidatedData.offeringDelivered =
+        OfferingDeliveryOptions.Onsite;
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+      // Assert
+      // calculatedDataTotalTransportationAllowance = 13 * 30 = 390
```

Comment:
I think it's not a good idea to hard comment those numbers. Instead we could write the logic:
`// calculatedDataTotalTransportationAllowance = dmnPartTimeProgramYearMaximums.limitTransportationAllowance * offeringWeeks`.

Reply:
This comment is updated to use `calculatedDataTotalTransportationAllowance = dmnPartTimeProgramYearMaximums.limitTransportationAllowance * offeringWeeks`. Please have a look.

Reply:
Please do the same for the other parts using fixed values in the tests.